Identity/IdentityImplementing.swift

```diff
@@ -25,6 +25,22 @@ public class IdentityImplementing: Identity {
         self.graphQLManager = graphQLManager
     }
 
+    public func getSSO(provider: String, completion: @escaping (Result<String?, Error>) -> Void) {
```

Comment:
Should return an SSO object, which contains the URL as a string.

Reply:
Reason being, if we ever need to return anything else other than the URL, we won't have to make a breaking change to the interface, and can just add it to the SSO object.

Reply:
Fixed it, thank you!